src/XliffTasks.Tests/VsctDocumentTests.cs

```diff
@@ -67,7 +67,7 @@ public void BasicLoadAndTranslate()
             document.Translate(translations);
             document.Save(writer);
 
-            Assert.Equal(expectedTranslation, writer.ToString());
+            AssertHelper.AssertWithoutLineEndingDifference(expectedTranslation, writer.ToString());
```

Comment:
I suggest AssertEx class name since we use that in lots of other dotnet projects for extra assert methods. And the name should have Equal in it still.

`AssertEx.EqualIgnoringLineEndings`